idaes/apps/flexibility_analysis/_check_dependencies.py

```diff
@@ -0,0 +1,21 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES).
+#
+# Copyright (c) 2018-2023 by the software owners: The Regents of the
+# University of California, through Lawrence Berkeley National Laboratory,
+# National Technology & Engineering Solutions of Sandia, LLC, Carnegie Mellon
+# University, West Virginia University Research Corporation, et al.
+# All rights reserved.  Please see the files COPYRIGHT.md and LICENSE.md
+# for full copyright and license information.
+#################################################################################
+"""
+This module is only used to check dependencies for unit tests.
+"""
+import unittest
```

Comment:
Seeing as this is just for testing, should it be moved into the `tests` folder?

Reply:
Great point. I'll move this.